shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1150,11 +1196,19 @@ - (instancetype)init {
 - (void)dealloc {
   [self hideTextInput];
   [_reusableInputView release];
+  [_inputHider release];
```

Comment:
there needs to be ` [_enableFlutterTextInputViewAccessibilityTimer release]` as well =)

Reply:
that will be trigger by hideTextInput, my thinking is that the timer is created in showtextinput, so it should be removed in hidetextinput

Reply:
Yea, I think in practice you are right since the timer will retain the FlutterTextInputPlugin until the timer fires.